app/styles/new-signup.css

```diff
@@ -0,0 +1,88 @@
+.container {
```

Comment:
Can we please use CSS variables for the colors ?

Reply:
okay I will do that